docs/development.md

```diff
@@ -237,7 +237,7 @@ Executing the tests is done via `./utils/test-cypress.sh`:
 ```
 The test_specifications which get executed are specified in `cypress.json`.
 
-More details on cypress-testing can be found in [cypress-testing.md](docs/cypress-testing.md).
+More details on cypress-testing can be found in [cypress-testing.md](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/docs/cypress-testing.md).
```

Comment:
For me this link does not work. Also i would definitely prefer to have relative links here as otherwise it would be quite inconsistent in the documentation-page:
https://deploy-preview-1742--specter-desktop-docs.netlify.app/development/
If a link breaks, i'd prefer it breaks on github rather than the documentation-pages.

Reply:
> For me this link does not work. Also i would definitely prefer to have relative links here as otherwise it would be quite inconsistent in the documentation-page: https://deploy-preview-1742--specter-desktop-docs.netlify.app/development/ If a link breaks, i'd prefer it breaks on github rather than the documentation-pages.

Sure I'll add relative links.